tests/installation/releasenotes.pm

```diff
@@ -23,7 +23,8 @@ sub run() {
     }
 
     # workaround for bsc#1014178
-    if (check_screen('zfcp-popup'), 0) {
+    wait_still_screen(5);
+    if (check_screen('zfcp-popup')) {
```

Comment:
Where does the 'alt-e' come from? Do you mean it immediately presses 'alt-o', this is not catched as the dialogue is not ready to accept that yet, then press 'alt-e'? I guess the `send_key 'alt-o';` should be guarded by

```perl
wait_screen_change { send_key 'alt-o' };
```